metadata-ingestion/source_docs/redshift.md

```diff
@@ -8,10 +8,11 @@ To install this plugin, run `pip install 'acryl-datahub[redshift]'`.
 
 ### Prerequisites
 
-This source needs to access system tables that require `superuser` permission; otherwise, it won't be able to see all schemas/tables. 
-To add a superuser or grant superuser permission, please refer to the [Superusers page](https://docs.aws.amazon.com/redshift/latest/dg/r_superusers.html).
-
-If you are unable to add superuser permissions, please ensure the user has SELECT privilege on [`SVV_TABLE_INFO`](https://docs.aws.amazon.com/redshift/latest/dg/r_SVV_TABLE_INFO.html) table.
+This source needs to access system tables that require extra permissions.
```

Comment:
Should we mention anything at all about the super user?

Reply:
The user won't be a super user, it just has full access to all the system tables but it won't even be able to query the tables and also should not be able to add/remove users

Reply:
I will add this from the Redshift documentation:
```
Giving a user unrestricted access to system tables gives the user visibility to data generated by other users. For example, STL_QUERY and STL_QUERYTEXT contain the full text of INSERT, UPDATE, and DELETE statements, which might contain sensitive user-generated data.
```